test/e2e/changedFilesUtil.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs').promises;
+
+const BASE_PATH = '/home/circleci/project/';
```

Comment:
Nit: perhaps we could make this more portable by using `path.reoslve(__dirname, '..', '..')` instead

Reply:
changes added :+1: